staker/block_validator.go

```diff
@@ -325,6 +343,27 @@ func NewBlockValidator(
 			PosInBatch: 0,
 		}
 	}
+	if config().Dangerous.Revalidation.Enable && config().Dangerous.Revalidation.StartBlock > 0 {
```

Comment:
Since `start-block` must be greater than zero, do we need `Enable` as well?

Reply:
Done.